accesshandler/pkg/config/providers.go

```diff
@@ -17,16 +17,18 @@ import (
 var Providers map[string]Provider
 
 type Provider struct {
-	ID       string
-	Type     string
-	Version  string
-	Provider providers.Accessor `json:"-"`
+	ID         string
+	Type       string
+	Version    string
+	Deprecated bool
```

Comment:
nitpick: Having `is` prefix to `bool` variables is usually considered a good practice. 
```suggestion
	IsDepricated bool
```